README.md

```diff
@@ -0,0 +1,61 @@
+README
+======
+
+Why do I need this?
+-------------------
+
+If you have a composer package that needs to know
+the location of the project it is included into or
+the location of the vendor directory.
+
+Can I do without?
+-----------------
+
+Normally if you need to know the vendor folder you
+whould check if your package's configured vendor folder
+exists. If it does, that is your vendor folder, if it
+does not, than your package is installed as dependency,
+thus the vendor folder is two folders up from your package
+root.
+
+When you need the project root however the best you can
+do without locating the composer.json file (which might
+have sideeffects) is just guess that the location of the
+vendor folder is left to the default and go up some foldes.
+
+So if you want to adhere to the user's configuration you
+need some information from composer, which is handed to
+you by this package.
```

Comment:
I find it hard to understand the information in those 3 blocks, not because it's complex but because of the way it's explained.